src/components/drop-down/drop-down.tsx

```diff
@@ -10,12 +11,34 @@ export interface DropDownInputProps {
 }
 
 export const DropDownInput: React.SFC<DropDownInputProps> = SBStateless(props => {
-  return (
-      <div data-automation-id="DROP_DOWN_INPUT" onClick={props.onClick} className="drop-down-input">
-          <span className="label">{props.selectedItem ? props.selectedItem.label : 'Default Text'}</span>
-          <CaretDown className="caret" />
-      </div>
-  );
+    return (
+        <div data-automation-id="DROP_DOWN_INPUT" onClick={props.onClick} className="drop-down-input">
+            <span className="label">{props.selectedItem ? props.selectedItem.label : 'Default Text'}</span>
+            <CaretDown className="caret" />
+        </div>
+    );
+}, style);
+
+export interface DropDownListProps {
+    open: boolean;
+    items?: DropDownItem[];
```

Comment:
The more i think about it, i have problems with having the value as a string and not an object. I know the spec says so, but from my experience with `<TreeView />`, keeping a reference to an object coming from `dataSource` as a 'selected item' (what does value mean anyway?) is easy and efficient.
What are your toughts, @AviVahl?